Source/WebCore/dom/Node.cpp

```diff
@@ -424,9 +424,6 @@ Node::~Node()
     ASSERT(!m_previous);
     ASSERT(!m_next);
 
-    if (auto* textManipulationController = document().textManipulationControllerIfExists(); UNLIKELY(textManipulationController))
```

Comment:
I think so, yes. But it's still an extra branch in the common case that we don't really need anymore because of WeakHashSet.

Reply:
Yeah, and I think this is good. I found that some of performance issue in VMEntryScope comes from the Vector destructor while it is almost always zero sized, and it turned out that Vector destructor bloated the code size of the VMEntryScope destructor. So, if there is a code which is not necessary, removing it would be beneficial for the performance even though it is mostly not taken :)